drivers/pds/api/defaultTemplates.go

```diff
@@ -48,14 +48,14 @@ func (ds *DefaultTemplates) ListResourceSettingTemplates() ([]pds.ModelsResource
 }
 
 // ListStorageOptionsTemplates func
-func (ds *DefaultTemplates) ListStorageOptionsTemplates() ([]pds.ModelsStorageOptionsTemplate, error) {
-	dsClient := ds.apiClient.DefaultTemplatesApi
+func (ds *DefaultTemplates) ListStorageOptionsTemplates(tenantID string) ([]pds.ModelsStorageOptionsTemplate, error) {
+	dsClient := ds.apiClient.StorageOptionsTemplatesApi
 	ctx, err := pdsutils.GetContext()
 	if err != nil {
 		log.Errorf("Error in getting context for api call: %v\n", err)
 		return nil, err
```

Comment:
Same here, just return the verbose error, I don't see the point of logging it, if you are most likely will be handling this error where you call this function anyways, could you please improve this code since you are already touching all these functions in this PR

Reply:
Done